pkgs/servers/zookeeper/default.nix

```diff
@@ -17,12 +17,15 @@ stdenv.mkDerivation rec {
     mkdir -p $out
     cp -R conf docs lib ${name}.jar $out
     mkdir -p $out/bin
-    cp -R bin/{zkCli,zkCleanup,zkEnv}.sh $out/bin
+    cp -R bin/{zkCli,zkCleanup,zkEnv,zkServer}.sh $out/bin
     for i in $out/bin/{zkCli,zkCleanup}.sh; do
       wrapProgram $i \
         --set JAVA_HOME "${jre}" \
         --prefix PATH : "${bash}/bin"
     done
+    substituteInPlace $out/bin/zkServer.sh \
+        --replace /bin/echo ${coreutils}/bin/echo \
+        --replace "/usr/bin/env bash" ${bash}/bin/bash
```

Comment:
Yes, I'm replacing the shebang, and no, it's not getting automatically replaced. I removed the replace here and reran the build:

```
$ nixos-rebuild --show-trace -I nixos=/root/nixpkgs/nixos/ -I nixpkgs=/root/nixpkgs/ build
...
building path(s) ‘/nix/store/3s4n8pq7jq2cjpzxi8304yw3hjvfbfas-zookeeper-3.4.9’
...
$ head -n 1 /nix/store/3s4n8pq7jq2cjpzxi8304yw3hjvfbfas-zookeeper-3.4.9/bin/zkServer.sh 
#!/usr/bin/env bash
```

Don't know why that is, haven't looked deeply into the zookeeper package, just wanted to minimally modify it to support exhibitor (which requires zkServer.sh).